MagicBytesValidator/Formats/Pgm.cs

```diff
@@ -1,17 +1,17 @@
-using MagicBytesValidator.Models;
-
 namespace MagicBytesValidator.Formats;
 
-public class Pgm : FileType
+/// <see href="https://en.wikipedia.org/wiki/List_of_file_signatures"/>
```

Comment:
You provide the link in all classes.
It would be enough for me if it were done centrally.